test/helpers/types/types.go

```diff
@@ -336,3 +349,40 @@ func (l Logs) NonEmpty() bool {
 	}
 	return len(l) > 0
 }
+
+func MockPod() *v1.Pod {
+	uuidStr := string(utils.GetRandomWord(16))
+	podName := string(utils.GetRandomWord(8))
+	namespace := string(utils.GetRandomWord(8))
+	fakePod := v1.Pod{
+		TypeMeta: metav1.TypeMeta{
+			Kind: "Pod",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      podName,
+			Namespace: namespace,
+			UID:       types.UID(uuidStr),
+		},
+		Spec: v1.PodSpec{},
+	}
+	return &fakePod
+}
+
+func MockEvent(ref *v1.ObjectReference, eventType, reason, message string) *v1.Event {
```

Comment:
Can we use naming conventions for constructors here?   maybe "NewMockEvent" or "NewTestEvent"

Reply:
Ok